pageserver/src/consumption_metrics.rs

```diff
@@ -88,22 +89,12 @@ pub async fn collect_metrics(
 
     let node_id = node_id.to_string();
 
-    // reminder: ticker is ready immediatedly
-    let mut ticker = tokio::time::interval(metric_collection_interval);
-
     loop {
-        let tick_at = tokio::select! {
-            _ = cancel.cancelled() => return Ok(()),
-            tick_at = ticker.tick() => tick_at,
-        };
+        let started_at = Instant::now();
 
         // these are point in time, with variable "now"
         let metrics = metrics::collect_all_metrics(&cached_metrics, &ctx).await;
 
```

Comment:
why did we have the check of metrics being empty and why it isn't relevant now ?

Reply:
@koivunej please answer (we coded this PR in screen-sharing, he said it's ok to go)

Reply:
That's a left over from earlier. I didn't remove it for because I forgot it.

Removing it now allows an indentation change and is more correct: we do not have tests for this situation ("draining pageserver out of tenants, sending 0 metrics and shutting down right after, on restart we recover without a warning") but if we were to have them, we would see that the at-startup rescheduling would log a warning by mistake, because it would think it has missed a lot of ticks basically.